src/components/GameInfoV2/index.tsx

```diff
@@ -0,0 +1,231 @@
+import React from 'react'
+
+import classNames from 'classnames'
+
+import fallbackCard from '@/assets/fallback_card.jpg?url'
+import * as Images from '@/assets/images'
+
+import Button from '../Button'
+import Sticker from '../Sticker'
+import styles from './GameInfoV2.module.scss'
+
+interface GameImageProps {
+  src?: string
+  alt?: string
+  fallback?: React.ReactNode
+  className?: string
+}
+
+interface EditorChoiceType {
+  isEditorChoice: boolean
+  year?: number
+  className?: string
+}
+
+interface SocialLinks {
+  IconButton: React.ElementType
+  url: string
+}
+
+interface GameInfoV2i18n {
+  title: string
+  version: string
+  earlyAccess: string
+  playerCount: string
+  addingText?: string
+  notAddedToLibrary?: string
+  playText?: string
+  blockchainLabel?: string
+}
+
+interface BlockchainIconProps {
+  icon: React.ElementType
+}
+
+export interface GameInfoV2Props {
+  title: string
+  version?: string
+  earlyAccess?: boolean
+  playerCount?: string
+  image?: {
+    src: string
+    alt: string
+    fallback?: React.ReactNode
+  }
+  blockchains?: Array<{
+    icon: React.ElementType
+    isHoverable?: boolean
+    name?: string
+  }>
+  showRemainingCount?: boolean
+  remainingCount?: string | number
+  socialLinks?: SocialLinks[]
+  editorChoice?: EditorChoiceType
+  actionButton?: React.ReactNode
+  className?: string
+  i18n?: GameInfoV2i18n
+}
+
+const GameImage: React.FC<GameImageProps> = ({
+  src,
+  alt,
+  fallback = (
+    <img
+      src={fallbackCard}
+      alt="Game cover fallback"
+      className={styles.fallbackImage}
+    />
+  ),
+  className
+}) => {
+  const [hasError, setHasError] = React.useState(false)
+
+  if (!src || hasError) {
+    return fallback ? (
+      <div className={classNames(styles.imageFallback, className)}>
+        {fallback}
+      </div>
+    ) : null
+  }
+
+  return (
+    <img
+      src={src}
+      alt={alt || 'Game cover'}
+      className={classNames(styles.image, className)}
+      onError={() => setHasError(true)}
+    />
+  )
+}
+
+const BlockchainIcon: React.FC<{
+  blockchain: BlockchainIconProps
+  className?: string
+}> = ({ blockchain, className }) => (
+  <div className={classNames(styles.blockchainIcon, className)}>
+    {React.createElement(blockchain.icon, {
+      className: styles.blockchainSvg
+    })}
+  </div>
+)
+
+const MAX_VISIBLE_BLOCKCHAINS = 5
+
+const GameInfoV2: React.FC<GameInfoV2Props> = ({
+  title,
+  version,
+  earlyAccess,
+  playerCount,
+  image,
+  blockchains = [],
+  showRemainingCount = true,
+  remainingCount = 9,
+  socialLinks,
+  editorChoice,
+  actionButton,
+  i18n,
+  className
+}): JSX.Element => {
+  const renderEditorChoice = () => {
+    if (!editorChoice?.isEditorChoice) return null
+
+    return (
+      <div className={classNames(styles.editorChoice, editorChoice.className)}>
+        <Images.EditorChoice />
+        Editor&apos;s Choice {editorChoice.year || new Date().getFullYear()}
+      </div>
+    )
+  }
```

Comment:
```suggestion
  let editorChoice = null
  if (editorChoice?.isEditorChoice){
    editorChoice = (
      <div className={classNames(styles.editorChoice, editorChoice.className)}>
        <Images.EditorChoice />
        Editor&apos;s Choice {editorChoice.year || new Date().getFullYear()}
      </div>
    )
  }
```

This is the way we usually do conditional rendering for components we don't want to break out for readability

Reply:
We may want to consider making this a separate component if we'll use it elsewhere

Reply:
This exactly design will live just here, checked with @iamdavid0091